src/Loader/AbstractWebpackLoader.php

```diff
@@ -199,6 +199,10 @@ protected function resolveLocation(string $fileName): int
             return Asset::LOGIN;
         }
 
+        if (stristr($fileName, '-customizer-preview')) {
```

Comment:
is there a case, where an asset is only build explicit for "-customizer-preview"? Or is an Script/Style which is "-customizer" always also showing in "customizer-preview"?

Reply:
No, the scripts loaded in the `customizer_preview_init` runs in frontend but only for the customizer context. 
That's the script which bind the functions that get the settings values from the customizer (backend) via PostMessage.

The customizer is backend while the customizer-preview is frontend (loaded in an iframe).
Would not make any sense to load a script in both context since they adhere to different concerns.

Reply:
ok! :)